data/data/bootstrap/systemd/units/approve-csr.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Approve CSRs during bootstrap phase
+Wants=bootkube.service
+After=bootkube.service
```

Comment:
Can we add a `BindsTo` to make sure this service stops as soon as bootstrapping is complete?

Reply:
From https://www.freedesktop.org/software/systemd/man/systemd.unit.html#BindsTo=
```
This not only means a unit bound to another unit that suddenly enters inactive state, but also one that is bound to another unit that gets skipped due to a failed condition check (such as ConditionPathExists=, ConditionPathIsSymbolicLink=, … — see below) will be stopped, should it be running. Hence, in many cases it is best to combine BindsTo= with After=.
```

I was a little concern that if bootkube stops/restarts _which it does sometimes when pulling images sometime or other transient errors_ that cause approve-csr to stop too... So didn't dive deep...


Will do some local testing on this, and add BindTo.

Reply:
Testing locally systemd doesn't restart the approve-csr service when bootkube service becomes active again after a failure, approve-csr remain inactive.